microservices-orders/get-kafka-streams-examples.sh

```diff
@@ -2,7 +2,7 @@
 
 # Compile java client code
 [[ -d "kafka-streams-examples" ]] || git clone https://github.com/confluentinc/kafka-streams-examples.git
-(cd kafka-streams-examples && git fetch && git checkout 5.4.0-post && git pull && mvn clean compile -DskipTests package)
+(cd kafka-streams-examples && git fetch && git checkout 5.4.x && git pull && mvn clean compile -DskipTests package)
```

Comment:
@ybyzek not complete sure about this... there is no 5.4.0-post as of yet, so we'll have to coordinate a change when release cut occurs, otherwise this can't be ran.  Or, is it valid to leave it at 5.4.x going forward?

Reply:
It depends.  

@mohnishbasha -- will this line be automatically changed to `5.4.0-post` when the RC is cut?  Or should we manually change it to `5.4.0-post` now?